### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by Tianhao-Gu**
add hs2 commit to test

### CODE REVIEW

**Comment by Tianhao-Gu** [comment]
@MrCreosote Do you mind taking a look?

File: .travis.yml [hunk 1]
```diff
@@ -33,9 +33,13 @@ install:
   - git clone https://github.com/kbase/jars
   - export JARSDIR=`pwd`/jars/lib/jars/
   - export HS_VER=version_1.0.0
+  - export HS_COMMIT=f90f1647456d2fd39c7c8b340e6c0727e306bba7
   - export LOG_COMMIT=b549c557e3c519e0a55eadf7863a93db25cd6806
   - git clone -b $HS_VER --single-branch https://github.com/kbase/handle_service2.git
-  - cd handle_service2/lib/
+  - cd handle_service2/
+  - git reset --hard $HS_COMMIT
```
Comment (MrCreosote) [inline .travis.yml]: Out of curiosity why a reset vs a checkout

Comment (Tianhao-Gu) [reply .travis.yml]: @MrCreosote No specific reasons. Want me to do a `checkout` instead?

Comment (MrCreosote) [reply .travis.yml]: Not really a big deal, but sure

Comment (Tianhao-Gu) [reply .travis.yml]: @MrCreosote cool. using checkout now.

**Pull request merged**